.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+text.md
```

Comment:
Dear @aima98 Your file seems perfect. Technically you have done it correctly. But you can still see that the `text.md` file is still not being ignored.
I tried to check that on my system. I didn't find a clear mistake BUT:
I found out what you might have done wrong: Kindly read [this](https://stackoverflow.com/questions/45400361/why-is-gitignore-not-ignoring-my-files)  
![image](https://user-images.githubusercontent.com/61361037/219044045-2013c285-162e-47aa-8565-121896f12cac.png)
I hope, this will resolve the issue. Good luck 👍